clash_royale_client.py

```diff
@@ -33,3 +33,7 @@ def get_player_info(self, player_tag):
         cleaned_tag = player_tag.replace('#', '').upper()
         request_url = '{}/players/%23{}'.format(self.base_url, cleaned_tag)
         return self.make_request(request_url, 'get_player_info')
+
+    # url param pre-conditoin -> clan_tag should not have # and all caps
+    def get_river_race_log(self, clan_tag):
+        return True

```

Comment:
Doesn't really matter but for empty functions yet to be implemented, you could do one of these instead
```
def get_river_race_log(self, clan_tag):
    # TODO
    pass
```
or
```
def get_river_race_log(self, clan_tag):
    # TODO
    return
```